cluster-scope/base/core/namespaces/aicos-osc-demo/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Kustomization
+apiVersion: kustomize.config.k8s.io/v1beta1
+namespace: aicos-osc-demo
+resources:
+    - namespace.yaml
+components:
+    - ../../../../components/project-admin-rolebindings/aicos-osc-demo
```

Comment:
Limit ranges and quotas are missing here. While you can have a namespace without quotas, I discourage it. Limit ranges are really good to have as well. 

See other namespaces for reference:
https://github.com/operate-first/apps/blob/master/cluster-scope/base/core/namespaces/apex/kustomization.yaml#L8-L9

Reply:
@tumido it make sense, however in this case we want to give developers bit of flexibility and potentially apply quotas subsequently. Also there is additional factor that that they will need GPU resources as well

Reply:
Ok then. If you as a cluster admin are fine with that, than sure.

Just keep in mind that once you decide to add `ResourceQuota`, you need to apply the `LimitRange` with pod defaults as well. Otherwise the quota calculations for pods without set `resources` (either request or limit for both CPU and memory) would fail and the pod would subsequently fail to schedule. Which is an interesting problem, imo.

https://github.com/operate-first/apps/issues/1142#issue-1020981783

Btw, GPUs can be specified in the quota as well (and if they are not, they are not restricted).